packages/js/product-editor/src/components/prepublish-sidebar/style.scss

```diff
@@ -0,0 +1,55 @@
+.woocommerce-product-publish-panel {
+	bottom: 0;
+	right: 0;
+	left: 0;
+	overflow: auto;
```

Comment:
We also want a `fixed` position to avoid it from scrolling with the page.

Reply:
https://github.com/woocommerce/woocommerce/assets/2240960/cbb090fc-6953-49fa-b02a-ea4aedb0148f